react-native/README.md

```diff
@@ -0,0 +1,54 @@
+
+We expect this project to take experienced developers about two to four hours to complete. We know not everyone has unlimited time to work on interview projects and we do not expect that everyone completes all of the requirements. However, you may be asked how you might have solved a requirement you missed had you been able to work on this project without time constraints. Additionally, if you want to extend the project beyond the listed requirements (e.g. completing some of the stretch goals, or being creative with your own requirements), please feel free to do so as you see fit. Please note that we prefer higher-code quality over additional features if you start to run out of time.
+
+Use of your favorite libraries, SDKs and frameworks is welcomed but optional. The project outlined below can be completed with or without these tools.
+
+Please be prepared to demo your solution in a working development environment on the day of your interview. When you present your craft demo, we will ask you to describe how you arrived at your solution. After that, we will ask for a tour of your code with a focus on how you arrived at the core functionality and any design decisions and/or trade-offs you made during the process.
+
+## User story
+Sally's garage manages a high volume of inventory for many independent auto mechanics using a Quick Base application. 
+Sally is hoping to use a native iOS application to add new models to their inventory tracking Quick Base app. 
+
+## Technical Requirements
+
+1. Clone our demo app from the App Exchange (see the [FAQ](#faq))
+1. Recreate a React native version of the `create new part` form and post the data via the quick base api to your app.
+    1. To get to the create new record view, go to `My Apps`
+    1. Under my apps, select `React Native Craft Demo`
+    1. After selecting your app, navigate to `Parts`
+    1. Click `+ New Part` 
+1. Create a `react-native` view that pushes data to the Quick Base API
+    1. When the user clicks the "Add" button on the form, a new record should be created in the Quick Base app in the parts tables with the data the user entered. **Hint: You can use one of our APIs to add a new record. See the [API Help documentation](https://help.quickbase.com/api-guide/index.html#add_record.html%3FTocPath%3DQuick%2520Base%2520API%2520Call%2520Reference%7C_____6)** 
+
+## User story for stretch goals
+Sally is hoping to be able to scan a barcode using our native application and have that data pushed into the "Barcode" 
+field in their Quick Base application. The mobile app should allow one of the floor managers to view the provided add 
+new record web page, and upon clicking the "Barcode" field, there should be a barcode scanner (fine to use QR codes 
+for testing) that scans and then returns that data to the add new record view.   
+
+## Stretch Goals
```

Comment:
Let's add a quick comment here. After moving the user story up, it looks like both of these need to be completed.

> If you have time, you can optionally complete one or more of the stretch goals listed below or come up with your own ideas for functionality you want to add to the app.